tests-system/rbt-signature-len-1/foo.rsl

```diff
@@ -0,0 +1,12 @@
+package Foo
+
+type T {
+    a String
+    b Integer
+    c Integer[1..*]
+}
+
+checks T {
+    len(a) > 7, "a must be longer than 7"
+    len(b) > 7, "b must be longer than 7"
```

Comment:
consider to add a check for `c` as well (but before the check for `b`), otherwise it is not parsed